README.rst

```diff
@@ -53,7 +53,7 @@ On Linux, see our `prerequisites <https://github.com/Azure/azure-cli/blob/master
 
 .. code-block:: console
 
-   $ pip install --user azure-cli
+   $ pip install --user --upgrade azure-cli
```

Comment:
@yugangw-msft yes it will
If you want to fix a version, use the ~= syntax for instance
pip install --user --upgrade azure-cli~=1.0.0
that will not install 2.x